grind/bin/grind

```diff
@@ -355,6 +355,10 @@ class TestRunner:
                             type=int,
                             default=default_timeout,
                             help="Per-task timeout. The default timeout is %s seconds." % default_timeout)
+        parser.add_argument('--java-version',
+                            type=int,
+                            default=7,
```

Comment:
Yes it should be in pconfig. Can we keep the flag for convenience?

BTW, have you tried [configglue](https://pythonhosted.org/configglue/intro/quickstart.html)? I found it very easy to use. It can automatically generate config file options, command line options, and env vars.